core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -181,25 +207,23 @@ static paddr_t core_mmu_get_ul1_ttb_pa(void)
 {
 	paddr_t pa = virt_to_phys((void *)core_mmu_get_ul1_ttb_va());
 
-	if (pa & ~TEE_MMU_TTB_UL1_MASK)
+	if (pa & ~TTB_UL1_MASK)
 		panic("invalid user l1 table");
 	return pa;
 }
 
-static void *core_mmu_alloc_l2(struct tee_mmap_region *mm)
+static void *core_mmu_alloc_l2(size_t size)
```

Comment:
I can't use the same approach, because there are other users of this function, and they depend on `size` parameter. 
I could inline it manually, but this is not best way. Actually, it would be better to align `core_mmu_lpae.c` implementation with current state of `core_mmu_v7` because it is better to have dedicated allocator function.